vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -12749,7 +12747,6 @@ unit:MDOLLAR-PER-FLIGHT
   a qudt:Unit ;
   qudt:expression "\\(\\(M\\$/Flight\\)\\)"^^qudt:LatexString ;
   qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T0D1 ;
```

Comment:
Another legacy from the NASA days! The URI should be changed to MegaDOLLAR-PER-FLIGHT, but the dimensionality is correct - a currency per a dimensionless unit (Flight).

Reply:
Should the URI be changed as part of this PR, or should I just add the quantity kind back?